From 4e2ba86e2b8d72964a52fdc984794ad4c0ef54af Mon Sep 17 00:00:00 2001
From: Kaige Fu <kaige.fu@intel.com>
Date: Thu, 21 Feb 2019 14:15:44 +0000
Subject: [PATCH 1343/1494] acrn: fix used uninitialized warning

This patch fix two used uninitialized warnings in acrn_hvlog.c and
acrn_trace.c

Change-Id: Ie29a4e26a4fb5e405b56ad96144dcfc612ec924d
Signed-off-by: Kaige Fu <kaige.fu@intel.com>
Reviewed-by: Yonghua Huang <yonghua.huang@intel.com>
---
 drivers/acrn/acrn_hvlog.c | 1 +
 drivers/acrn/acrn_trace.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/acrn/acrn_hvlog.c b/drivers/acrn/acrn_hvlog.c
index 65fbceb..da04aeb 100644
--- a/drivers/acrn/acrn_hvlog.c
+++ b/drivers/acrn/acrn_hvlog.c
@@ -353,6 +353,7 @@ static int __init acrn_hvlog_init(void)
 		return 0;
 	}
 
+	memset(&hw_info, 0, sizeof(struct acrn_hw_info));
 	ret = hcall_get_hw_info(virt_to_phys(&hw_info));
 	if (!ret)
 		pcpu_nr = hw_info.cpu_num;
diff --git a/drivers/acrn/acrn_trace.c b/drivers/acrn/acrn_trace.c
index 852c8b0..010acb9 100644
--- a/drivers/acrn/acrn_trace.c
+++ b/drivers/acrn/acrn_trace.c
@@ -185,6 +185,7 @@ static int __init acrn_trace_init(void)
 		return -EINVAL;
 	}
 
+	memset(&hw_info, 0, sizeof(struct acrn_hw_info));
 	ret = hcall_get_hw_info(virt_to_phys(&hw_info));
 	if (!ret)
 		pcpu_num = hw_info.cpu_num;
-- 
2.7.4

